packages/components/src/dropdown/index.js

```diff
@@ -110,7 +113,9 @@ export default function Dropdown( props ) {
 					// This value is used to ensure that the dropdowns
 					// align with the editor header by default.
 					offset={ 13 }
-					anchorRef={ ! hasAnchorRef ? containerRef : undefined }
+					anchor={
+						! hasPopoverAnchor ? containerRef.current : undefined
```

Comment:
Not an issue with this PR, but I'm not 100% sure what this means in practice 🤔 "If you pass me a `popoverProps.anchor*`, I'll ignore it but pass my own anchor"? Why not just always pass the `containerRef.current`? And when we add TS types to this component, should `popoverProps.anchor*` stuff be omitted?

Reply:
The idea is:

"If `popoverProps` doesn't contain any anchor for the `Popover`, then I will set my own (containerRef.current)". Since `popoverProps` are passed to the `Popover` a few lines after this comment, whatever anchor is set in them will be passed to `Popover`

The logic is a bit more complicated because an anchor can be currently passed in multiple different ways. But after we deprecate all other anchor props and we'll be left just with `anchor`, this logic won't be necessary: we could just always pass `anchor`, and if `popoverProps` contains an `anchor`, it will override the default one.

Makes sense?

Reply:
> And when we add TS types to this component, should popoverProps.anchor* stuff be omitted?

According to the reasoning from my previous message, it should _not_ be omitted — an `anchor` passed through `popoverProps` should override the one set internally by `Dropdown` as a fallback

Reply:
Ah, gotcha, I read it the other way around 🙈 The `undefined` threw me off. Thanks for explaining!

Reply:
> Ah, gotcha, I read it the other way around 🙈 The `undefined` threw me off. Thanks for explaining!

It's definitely not the most straightforward logic 😅